thoth/storages/graph/postgres.py

```diff
@@ -1502,7 +1507,84 @@ def sync_analysis_result(self, document: dict) -> None:
 
     def sync_package_analysis_result(self, document: dict) -> None:
         """Sync the given package analysis result to the graph database."""
-        raise NotImplementedError
+        package_analysis_document_id = PackageAnalysisResultsStore.get_document_id(document)
+        package_name = document["metadata"]["arguments"]["python"]["package_name"]
+        package_version = document["metadata"]["arguments"]["python"]["package_version"]
+        index_url = document["metadata"]["arguments"]["python"]["index_url"]
+
+        try:
+            with self._session.begin(subtransactions=True):
+                python_package_index, _ = PythonPackageIndex.get_or_create(
+                    self._session,
+                    url=index_url,
+                )
+                python_package_version_entity, _ = PythonPackageVersionEntity.get_or_create(
+                    self._session,
+                    package_name=package_name,
+                    package_version=package_version,
+                    python_package_index_id=python_package_index.id,
+                )
+                package_analyzer_run, _ = PackageAnalyzerRun.get_or_create(
+                    self._session,
+                    package_analyzer_name=document["metadata"]["analyzer"],
+                    package_analyzer_version=document["metadata"]["analyzer_version"],
+                    package_analysis_document_id=package_analysis_document_id,
+                    datetime=document["metadata"]["datetime"],
+                    debug=document["metadata"]["arguments"]["thoth-package-analyzer"]["verbose"],
+                    package_analyzer_error=document["result"].get("error", False),
```

Comment:
@fridex It seems we also added an attribute called `package_analyzer_error_message` which should come from `document["result"].get("error_message", "")`